src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -295,6 +305,14 @@ class ScallopGobraConfig(arguments: Seq[String], isInputOptional: Boolean = fals
   }
   validateInput(input, module, include)
 
+  // cache file should only be usable when using viper server
+  validateOpt (backend, cacheFile) {
+    case (Some(ViperBackends.ViperServerWithSilicon), Some(_)) => Right()
+    case (Some(ViperBackends.ViperServerWithCarbon), Some(_)) => Right()
+    case (_, None) => Right()
+    case (_, Some(_)) => Left("Cache file option only works with viper server as a backend")
```

Comment:
Would change to "Cache file can only be specified when the backend uses Viper Server"